setup/setup.sh

```diff
@@ -52,19 +61,14 @@ echo "Setup the surrounding softwate and environment"
 echo
 echo "Start up MySQL for database access"
 oc project $PROD_PROJECT
-oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/mysql-ephemeral-template.json
-oc new-app --template=mysql-ephemeral --param=MYSQL_PASSWORD=password --param=MYSQL_USER=dbuser --param=MYSQL_DATABASE=sampledb
+oc new-app mysql-ephemeral --param=MYSQL_PASSWORD=password --param=MYSQL_USER=dbuser --param=MYSQL_DATABASE=sampledb
 
 echo "Start up Broker"
 oc import-image amq62-openshift --from=registry.access.redhat.com/jboss-amq-6/amq62-openshift --confirm
-oc create -f projecttemplates/amq62-openshift.json
-oc new-app --template=amq62-basic --param=MQ_USERNAME=admin --param=MQ_PASSWORD=admin --param=IMAGE_STREAM_NAMESPACE=$PROD_PROJECT
-
-oc policy add-role-to-user edit system:serviceaccount:${CICD_PROJECT}:jenkins -n ${PROD_PROJECT}
-oc policy add-role-to-user edit system:serviceaccount:${CICD_PROJECT}:default -n ${PROD_PROJECT}
-oc policy add-role-to-user view --serviceaccount=default -n ${PROD_PROJECT}
-
-
-
+oc new-app -f projecttemplates/amq62-openshift.json --param=MQ_USERNAME=admin --param=MQ_PASSWORD=admin --param=IMAGE_STREAM_NAMESPACE=$PROD_PROJECT
 
 
+oc policy add-role-to-user edit system:serviceaccount:${DEV_PROJECT}:jenkins -n ${PROD_PROJECT}
+oc policy add-role-to-user edit system:serviceaccount:${DEV_PROJECT}:default -n ${PROD_PROJECT}
+oc policy add-role-to-user system:image-puller system:serviceaccount:${PROD_PROJECT}:default -n ${DEV_PROJECT}
+oc policy add-role-to-user view --serviceaccount=default -n ${DEV_PROJECT}
```

Comment:
How about switching back to the first project at the end, where the user should typically be to continue?

`oc project $DEV_PROJECT`

Reply:
edited